sdk/core/core-amqp/src/connectionConfig/connectionConfig.ts

```diff
@@ -156,12 +153,6 @@ export const ConnectionConfig = {
     }
     config.endpoint = String(config.endpoint);
 
-    if (config.useDevelopmentEmulator && !isLoopbackAddress(config.endpoint)) {
```

Comment:
Is this expected?
"Other Changes" subheading in the changelog for 4.3.0 says otherwise.

Reply:
Yes, we were validating if we were trying to connect to the emulator at the localhost only but this is no longer a requirement. See the PR description for context.